src/tools/compiletest/src/common.rs

```diff
@@ -118,6 +118,14 @@ impl fmt::Display for PassMode {
     }
 }
 
+/// Stage where an RMC test should fail.
+#[derive(Copy, Clone, Debug, PartialEq, PartialOrd)]
+pub enum RMCFailMode {
+    Check,
+    Codegen,
+    Verify,
+}
```

Comment:
Can you add a comment explaining what is tested for each one of the? I had trouble understanding the `Check` one so I think we should be clear about that.